src/components/presentational/Switch/Switch.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import "./Switch.css"
+
+export interface SwitchProps {
+	enabled: Boolean;
```

Comment:
Ah, that's a great point.  I will change it to "on"

I think there's no point in actually setting a "disabled" state since the onClick needs to be handled by the parent anyways (which would be setting the disabled state)...so it could just ignore the event.  Only reason I would add is if we were going to visually style the switch differently if disabled